### DISCUSSION TITLE
Can't to request to API, but in local and another service for test api all work like need

### DISCUSSION THREAD

**Discussion by tamerlansarakaev** [vercel/community#5872] [Help]
### Summary

Hello, I have a problem when I try to use a third-party anilibria api

### Steps to Reproduce

It gives a 403 Forbidden error, or 500, although on the local and also on third-party services for checking the api it produces the required result. The name of the project where the tanime-api error occurs: Error fetching anime data: SyntaxError: Unexpected token < in JSON at position 0
     at JSON.parse (<anonymous>)
     at parseJSONFromBytes (node:internal/deps/undici/undici:4747:19)
     at successSteps (node:internal/deps/undici/undici:4718:27)
     at fullyReadBody (node:internal/deps/undici/undici:1433:9)
     at processTicksAndRejections (node:internal/process/task_queues:95:5)
     at specConsumeBody (node:internal/deps/undici/undici:4727:7)
     at AnimeService.getAnimeWithCode (/vercel/path0/src/anime/anime.service.ts:11:26)
     at /var/task/node_modules/@nestjs/core/router/router-execution-context.js:46:28
     at /var/task/node_modules/@nestjs/core/router/router-proxy.js:9:17

**Comment by amyegan** [comment]
Hey @tamerlansarakaev. It sounds like the error is coming from the third-party API returning an HTML response instead of a JSON string. This is often caused by third-party providers if they either refuse the request because your app is not authorized correctly (401/403), because they have some issues (5xx) or if the requested resource couldn't be found (404). 

Please check that the correct URL and all required tokens are included as [environment variables](https://vercel.com/docs/projects/environment-variables) and are used when calling the third-party API. Missing or expired tokens are a common cause of this type of error. 🙏
